geoportailv3/static/js/mymaps/mymaps.html

```diff
@@ -50,43 +50,82 @@
 </div>
 
 <!-- Modification Modal -->
-<ngeo-modal ngeo-modal-shown="ctrl.modifying">
+<ngeo-modal ngeo-modal-shown="ctrl.modifying || ctrl.creating">
```

Comment:
I don't understand why you want to do that, but I may not have the full context.

To me the question is: do you need to define a specific action when the user closes the modal? For example, do you need to reset `modifying` or `creating` (or both) to `false`?